File: src/javascript/starpattern.js
function letDraw(i){
	let str = ''; r = 1;
	switch (i) {
		case 1:
			str = "*       *";
			break;
		case 2:
			str = "* *   *";
			break;
		default:
			str = "* ";
			r = i
			break;
	}
	console.log(str.repeat(r))
}

function starpattern(n) {
	for(let i=0; i<=n; i++) {
		letDraw(i)
	}
}

starpattern(5)

/*
 starpattern(5)
 
*       *
* *   *
* * * 
* * * * 
* * * * *
 
 */

File: src/javascript/starpattern.java
public class Starpattern
{ 
    public static void pattern(int n) 
    { 
        int i, j;  
        for(i=0; i<n; i++)              
        {
        for(j=2*(n-i); j>=0; j--)        
            {           
                System.out.print(" ");     
            } 
            for(j=0; j<=i; j++)          
            {       
                System.out.print("* ");     
            }           
            System.out.println();        
        } 
    } 
    
    public static void main(String args[]) 
    { 
        int n = 5; 
        pattern(n); 
    } 
}


/*
 *
 * *
 * * *
 * * * *
 * * * * *
 */
